jjb/device/device-grove-c.yaml

```diff
@@ -30,3 +30,7 @@
           build-node: ubuntu18.04-docker-arm64-4c-2g
           docker_build_args: '-f scripts/Dockerfile.alpine-3.9'
           docker_name: docker-device-grove-c-arm64
+      - '{project-name}-{stream}-stage-docker-arm':
```

Comment:
Is there any intent to include non-arm builds? The list that Jim White provided me with for release included non-arm images for device-grove-c which are not actually being built.

Reply:
No, device-grove-c only supports the Raspberry Pi 3+ which is arm only. So we are only going to have arm images for this repo.